app/controllers/votes_controller.rb

```diff
@@ -1,2 +1,32 @@
 class VotesController < ApplicationController
+  before_action :authenticate_user!
+  
+  helper_method :vote
+
+  def create
+    vote = current_user.votes.build(vote_params)
+    vote.save
```

Comment:
I would skip creating the local var here.

Reply:
Thanks. I had an earlier version where I wanted some attribute from `vote` to inform the redirect path, but its not needed now.